test/extensions/email_confirmation/ecto/schema_test.exs

```diff
@@ -62,6 +62,12 @@ defmodule PowEmailConfirmation.Ecto.SchemaTest do
     refute changeset.errors[:unconfirmed_email]
   end
 
+  test "changeset/2 doesn't accept when email already taken by another user" do
```

Comment:
Should also test `RepoMock.update/1` with valid email change to ensure the prepare callback works as exepected.